test/functional/notes_controller_test.rb

```diff
@@ -865,6 +865,26 @@ def test_get_rss_feed
     end
    end
 
+   test "drafts are sorted by last updated" do
+    UserSession.create(users(:jeff))
+    saved, node, revision = Node.new_note( title: 'Sort Drafts', draft: 'true', uid: 2, body:'I have sorted drafts' )
+    revision_timestamp = revision.timestamp
+
+    assert_equal 3, node.status
+    assert_not_nil revision_timestamp
+
+    Timecop.freeze(Date.today + 1.day) do
+      get :publish_draft, params: { id: node.id }
+
+      assert_response :redirect
+      node = assigns(:node)
+      assert_equal 1, node.status
+    end
+    
+    assert_not_equal revision_timestamp, node.latest['timestamp']
```

Comment:
Here could we assert what it should be equal to? Also, we could assert the wrong (original) timestamp before we publish the draft, to show that it was different before vs after. 

This looks great! Thanks Tilda!!!

Reply:
@jywarren I've made the changes